restapi/tags.go

```diff
@@ -0,0 +1,20 @@
+package restapi
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"github.com/uwblueprint/shoe-project/restapi/rest"
+)
+
+func (api api) ReturnAllUniqueTags(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var tags []models.Tag
+
+	err := api.database.Distinct("name").Find(&tags).Error
```

Comment:
For right now you can return just the tag names (i.e dont need stories). It doesn't make sense to return stories as several stories can have the same tag